capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
- [ ] Please be mindful that the `upcase` method capitalizes all of the passed string and not just the first letter as [required](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Implement%20a%20method%20correct_name%20that%20capitalizes%20the%20output%20of%20%40nameable.correct_name.).
### Hint: You can use the `capitalize` method.

Reply:
I know it does, and that's what the instruction required.